cmd/kube-proxy/app/server.go

```diff
@@ -477,6 +477,11 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 			return nil, fmt.Errorf("unable to read IPTables MasqueradeBit from config")
 		}
 
+		nodeIP, err := getNodeIP(client, hostname)
+		if err != nil {
+			return nil, fmt.Errorf("unable to fetch node IP: %v", err)
```

Comment:
Actually, I have to admit I am not familiar with this concept of a standalone kube-proxy.  Where can I learn about it?  I thought the proxy was all about supporting Service objects --- which it learns about from the apiserver.